app/views/items/_items_images_component.html.erb

```diff
@@ -1,85 +1,65 @@
 <% if defined?(width) && defined?(height) %>
-<% image_style="width: #{width}; height: #{height}; object-fit: cover;" %>
-  <% else %>
-    <% image_style="" %>
+  <% image_style="object-fit: cover; width: 100%; height: 300px;" %>
```

Comment:
In der akutellen Version werden die übergebenen Parameter (height & width) gar nicht verwendet. Dadurch sind die Bilder auf der show-Seite auch bei sehr großen Bildschirmen mit 300px relativ klein.

Ich hab es nicht gegen alle Edge-cases getestet aber vielleicht kannst du den height-Parameter wieder mit einbauen (#{height};)? So oder so, bitte alle unbenutzten Parameter (in dem Fall dann eben width) hier und auch in allen Referenzen auf diesen Componenten entfernen.